enarx-keep-sgx/src/main.rs

```diff
@@ -142,9 +145,27 @@ fn main() {
     //   2. Asynchronous exits (AEX) are handled here to minimize the amount
     //      of assembly code used.
     loop {
-        match enclave.enter(&mut block as *const _ as _, 0, 0, Leaf::Enter, 0, 0) {
+        match enclave.enter(&mut block as *const _ as _, 0, 0, leaf, 0, 0) {
             // On InvalidOpcode: re-enter the enclave with EENTER (CSSA++).
-            Err(Some(ei)) if ei.trap == Exception::InvalidOpcode => (),
+            Err(Some(ei)) if ei.trap == Exception::InvalidOpcode => {
+                leaf = Leaf::Enter;
+                cssa += 1;
+            }
```

Comment:
```suggestion
            Ok(_) if unsafe { block.msg.req.num } == ERESUME.into() => Leaf::Resume,
            Ok(_) => { block.msg.rep = unsafe { block.msg.req.syscall() }; Leaf::Enter },
            Err(Some(ei)) if ei.trap == Exception::InvalidOpcode => Leaf::Enter,
```

Something like that to show the clear options. I don't care if you reintroduce `cssa` if it is used in debugging output. But it is currently unused.